manifests/params.pp

```diff
@@ -57,5 +57,10 @@
 
     $listen_on_v6         = 'any'
 
+    $recursion            = 'yes'
     $allow_recursion      = []
+    $allow_query          = [ 'any' ]
+
+    $dnssec_enable        = 'no'
+    $dnssec_validation    = 'no'
```

Comment:
http://ftp.isc.org/isc/bind9/cur/9.8/doc/arm/Bv9ARM.ch06.html states the default is ```yes``` and I'd prefer to match defaults.